src/client/main.scss

```diff
@@ -66,3 +66,9 @@ svg {
 //  background: $brand;
 //  opacity: 0.2;
 //}
+
+:root {
+  @each $var, $val in $theme-colors {
```

Comment:
I'd probably go with `$theme-variables`, but that's rename so let's resolve it later.

Reply:
Or maybe not. I'm not sure now. Just don't be surprised if I change it in few months ;)